javers-spring/src/main/java/org/javers/spring/DefaultAuthorProvider.java

```diff
@@ -0,0 +1,11 @@
+package org.javers.spring;
+
+/**
+ * @author Pawel Szymczyk
+ */
+public class DefaultAuthorProvider implements AuthorProvider {
+    @Override
+    public String provide() {
+        return "author";
```

Comment:
może tu lepiej by pasowało coś w stylu "undefined", żeby user wiedział ze ma to uzupełnić